tests/mcp/test_hunt_tools.py

```
@@ -0,0 +1,169 @@
+
+@pytest.fixture
+def workspace(tmp_path):
+    return _setup_workspace(tmp_path)
+
+
+@pytest.fixture
+def server(workspace):
+    return create_server(str(workspace))
+
+
+def _call_tool(server, tool_name, arguments=None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_call_tool`

(ANN202)